src/notifications/TwitchNotifier.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const moment = require('moment');
+const TwitchEmbed = require('../embeds/TwitchEmbed');
+const Broadcaster = require('./Broadcaster');
+const { platforms } = require('../CommonFunctions');
+const logger = require('../Logger');
+
+/**
+ * Watches for Twitch go-lives and broadcasts them
+ */
+class TwitchNotifier {
+  constructor({
+    client, settings, messageManager,
+  }) {
+    this.broadcaster = new Broadcaster({
+      client,
+      settings,
+      messageManager,
+    });
+
+    this.lastStartedAtTime = null;
+    this.user = process.env.TWITCH_USER_LOGIN || 'warframe';
+
+    // save this in closure for setInterval
+    const self = this;
+    setInterval(async () => { await self.pollTwitch(); },
```

Comment:
sorry, late noticing this. since it's passing the function reference, should be able to pass it as `setInterval(this.pollTwitch, process.env.TWITCH_POLL_INTERVAL_MS || 60000);`
or if it needs to be bound... `setInterval(this.pollTwitch.bind(this), process.env.TWITCH_POLL_INTERVAL_MS || 60000);` since the first param is just a function of any type, doesn't have to directly be anonymous inside, and we don't care if the setInterval treats it as async or not, cause it'll execute and handle logic for errors and whatnot on it's own, theoretically....

Reply:
done!